### PR TITLE
Kibana Attempt 2

### PR DESCRIPTION
**Pull request by AWare**
Reverts guardian/membership-frontend#1695

Attempting to make this right.

**Changes since Reverts guardian/membership-frontend#1694**
Config file on server already has kibana details in, so no need to use user data.

### CODE REVIEW

File: cloud-formation/membership-app.cf.yaml [hunk 1]
```diff
@@ -107,6 +113,13 @@ Resources:
             aws --region ${AWS::Region} s3 cp s3://membership-dist/${Stack}/${Stage}/frontend/frontend_1.0-SNAPSHOT_all.deb /tmp
             mkdir /etc/gu
             aws --region ${AWS::Region} s3 cp s3://membership-private/${Stage}/membership.private.conf /etc/gu
+             cat <<EOF >>/etc/gu/subscriptions-frontend.private.conf
```
Comment (jacobwinch) [inline cloud-formation/membership-app.cf.yaml]: Should this be membership.private.conf?

**Comment by jacobwinch** [comment]
What's the difference between this PR and the original attempt?

**Comment by johnduffell** [comment]
@jacobwinch the changes to the config are no longer happening in the userdata in the cloudformation courtesy of @davidfurey suggestion this morning.  They are in the S3 config.

**Comment by johnduffell** [comment]
@pvighi has created a code environment so it would be worth running it there first.

**Comment by jacobwinch** [comment]
Thanks @johnduffell - this happened after I asked the original question, which is why I was confused!

**Comment by pvighi** [comment]
@johnduffell I think this is the one project I didn't create a CODE env for since the template was already in yaml and updating with amigo so I didn't need to change anything.

**Pull request merged**